spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-cloudfoundry/src/main/java/org/springframework/cloud/data/module/deployer/cloudfoundry/GetApplicationStatisticsResponse.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.data.module.deployer.cloudfoundry;
+
+import java.util.HashMap;
+
+final class GetApplicationStatisticsResponse extends HashMap<String, ApplicationInstanceStatus> {
```

Comment:
In this case the response *is* a map, and I have no information about the keys, so I can't make it a field in the response object. What do you suggest?  I would like to say `GetApplicationStatisticsResponse = HashMap<...>`, but that's not allowed in Java.  `extends` is all I have left (and it works).